From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: NameDoesCode <donkeyfredward@gmail.com>
Date: Fri, 26 Mar 2021 23:38:42 -0400
Subject: [PATCH] Lag assist crash fix


diff --git a/src/main/java/net/minecraft/server/level/RegionLimitedWorldAccess.java b/src/main/java/net/minecraft/server/level/RegionLimitedWorldAccess.java
index 04006caeeb42b523d986efc313828557854718d7..b18c69149b5342fe82e22ddff351004ce884c64a 100644
--- a/src/main/java/net/minecraft/server/level/RegionLimitedWorldAccess.java
+++ b/src/main/java/net/minecraft/server/level/RegionLimitedWorldAccess.java
@@ -110,7 +110,7 @@ public class RegionLimitedWorldAccess implements GeneratorAccessSeed {
     @Override
     public IChunkAccess getChunkAt(int i, int j, ChunkStatus chunkstatus, boolean flag) {
         IChunkAccess ichunkaccess;
-
+        if (i > 134217000 || i < -134217000 || j > 134217000 || j < -134217000) return null;
         if (this.isChunkLoaded(i, j)) {
             int k = i - this.n.x;
             int l = j - this.n.z;
